app/src/main/java/com/aptoide/uploader/apps/InstalledApp.java

```diff
@@ -1,39 +1,217 @@
 package com.aptoide.uploader.apps;
 
 import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
 import android.os.Build;
+import android.os.Environment;
+import androidx.annotation.NonNull;
 import androidx.annotation.RequiresApi;
+import androidx.room.Entity;
+import androidx.room.Ignore;
+import androidx.room.PrimaryKey;
+import java.io.File;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+@Entity(tableName = "installed") public class InstalledApp {
+  @Ignore public static final int STATUS_COMPLETED = 4;
+  @PrimaryKey @NonNull private String packageAndVersionCode;
+  private String packageName;
+  private String name;
+  private String versionName;
+  private int versionCode;
+  private boolean isSystemApp;
+  private String apkPath;
+  private String iconPath;
+  private long installedDate;
+  @Ignore private Obb obbMain;
+  @Ignore private Obb obbPatch;
```

Comment:
Why are we ignoring obb's? Won't this fail the upload obb's after the session of the insertion, since it doesn't persist?

Reply:
Didn't really thought about it but it will probably fail when trying to upload. It was ignored because we needed a TypeConverter since its an Obb class.
I'll implement the converter so we can save the obbs in the db.